tests/suites/os/tests/config-json/index.js

```diff
@@ -385,11 +385,11 @@ module.exports = {
             )
         );
         test.comment(`Device should show previous boot records, showed ${testcount} boots`);
```

Comment:
I think we can remove this test.comment now, it was just a placeholder while the test was disable

Reply:
Roger